bpf/init.sh

```diff
@@ -275,50 +275,3 @@ if [ "${TUNNEL_PROTOCOL}" != "<nil>" ]; then
 	sed -i '/^#.*ENCAP_IFINDEX.*$/d' $RUNDIR/globals/node_config.h
 	echo "#define ENCAP_IFINDEX $ENCAP_IDX" >> $RUNDIR/globals/node_config.h
 fi
-
-if [ "$MODE" = "direct" ] || [ "$NODE_PORT" = "true" ] ; then
-	if [ "$NATIVE_DEVS" == "<nil>" ]; then
```

Comment:
can't comment above the changes, but since we're no longer referencing `NATIVE_DEVS` can we remove this arg?

Reply:
Ah yes, I'll comment it out as we did with a few other until complete removal of `init.sh`